uk.co.compucorp.civicrm.hrcomments/CRM/HRComments/BAO/Comment.php

```diff
@@ -58,7 +58,7 @@ public static function validateParams($params) {
    * @throws \CRM_HRComments_Exception_InvalidCommentException
    */
   public static function validateCommentSoftDeleteDuringUpdate($params) {
-    if (isset($params['id']) && $params['is_deleted'] == 1) {
+    if (isset($params['id']) && isset($params['is_deleted']) && $params['is_deleted'] == 1) {
```

Comment:
this `isset($params['is_deleted']) && $params['is_deleted'] == 1` can be replaced with `if(!empty($params['is_deleted']))`